creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -3,6 +3,7 @@ const axios = require('axios')
 const _ = require('lodash')
 const retry = require('async-retry')
 
+const config = require('./../config')
```

Comment:
no way for you to have known this but in this file we pass in `nodeConfig` to the class constructor so you can remove this import and instead pull the value from there 🙂

Reply:
thanks, I pushed a change for that